addons/account_edi/wizard/account_move_send.py

```diff
@@ -9,18 +9,13 @@ class AccountMoveSend(models.Model):
     def _get_mail_attachment_from_doc(self, doc):
         attachment_sudo = doc.sudo().attachment_id
         if attachment_sudo.res_model and attachment_sudo.res_id:
-            return [{
-                'id': attachment_sudo.id,
-                'name': attachment_sudo.name,
-                'mimetype': attachment_sudo.mimetype,
-                'placeholder': False,
-            }]
-        return []
+            return attachment_sudo
+        return self.env['ir.attachment']
 
-    def _get_default_email_attachment_data(self, mail_template, move):
-        """ Returns all the placeholder data and mail template data
+    def _get_invoice_extra_attachments(self, move):
```

Comment:
extends `_get_invoice_extra_attachments_data` instead because you need to add `'placeholder': False`

Reply:
`placeholder` is already set to `False` when parsing attachments from `_get_invoice_extra_attachments`
https://github.com/odoo/odoo/blob/21586fc2c175d7a5f14b4ff686fae6725d18092f/addons/account/wizard/account_move_send.py#L139